testsuite/basic/src/test/java/io/smallrye/jwt/config/JWTAuthContextInfoProviderTest.java

```diff
@@ -41,13 +41,12 @@ public void testDefaultGetOptionalContextInfo() {
         JWTAuthContextInfoProvider provider = JWTAuthContextInfoProvider.createWithKeyLocation("NONE", TEST_ISS);
         Optional<JWTAuthContextInfo> info = provider.getOptionalContextInfo();
         assertNotNull(info);
-        assertTrue(!info.isPresent());
+        assertTrue(info.isPresent());
     }
 
-    @Test(expected = IllegalStateException.class)
     public void testDefaultGetContextInfo() {
```

Comment:
@sberyozkin - I think `@Test` is missed here.

Reply:
@MikeEdgar I wonder why it passed so easily :-)